echo-webhooks/src/test/java/com/netflix/spinnaker/echo/scm/GithubWebhookEventHandlerTest.java

```diff
@@ -0,0 +1,116 @@
+package com.netflix.spinnaker.echo.scm;
+
```

Comment:
No copyright on this file - @dbyron-sf is that required?

Reply:
I can add one